modules/apps/object/object-api/src/main/java/com/liferay/object/system/SystemObjectDefinitionManager.java

```diff
@@ -49,11 +49,11 @@ public BaseModel<?> getBaseModelByExternalReferenceCode(
 			String externalReferenceCode, long companyId)
 		throws PortalException;
 
-	public String getExternalReferenceCode();
-
-	public String getExternalReferenceCode(long primaryKey)
+	public String getEntryExternalReferenceCode(long primaryKey)
```

Comment:
I saw that the term used is **BaseModel** instead of Entry. How about renaming to getBaseModelExternalReferenceCode?